.gitignore

```diff
@@ -25,8 +25,7 @@ _build
 _generated
 docs/api
 docs/generated
-docs/visualization/ngc6976.jpeg
-docs/visualization/ngc6976-default.jpeg
+docs/visualization/ngc6976*.jpeg
```

Comment:
We should avoid generating random files in the source tree (that's something I wanted to fix long ago, and forgot about it). Saving those files doesn't really add value to the docs, so I would be in favor of removing the `filename=...` arguments below.